Code/Engine/Foundation/System/MiniDumpUtils.h

```diff
@@ -15,6 +15,9 @@ extern "C"
 /// \brief Functionality for writing process mini-dumps (callstacks, memory snapshots, etc)
 struct EZ_FOUNDATION_DLL ezMiniDumpUtils
 {
+  /// \brief Checks if the MiniDumpUtils are supported on the current platform
+  static bool IsSupported();
```

Comment:
Since this is a compile time thing, should it maybe rather be a #define ? E.g. EZ_SUPPORTS_CRASHDUMPS ?

Could wrap the entire ezMiniDumpUtils class in that then.